### DISCUSSION TITLE
How to deploy if multiple constructs are there?

### DISCUSSION THREAD

**Discussion by kamalnathlp** [getlift/lift#212] [Constructs]
constructs:
  trackuserjobs:
    type: queue
    worker:
      handler: handler.consumer
      package:
        exclude:
          - ./**
        include:
          - "handler.js"
          - "logger.js"
      layers:
        - !Ref SqsWorkerLibLambdaLayer
    trackuserjobs1:
    type: queue
    worker:
      handler: handler.consumer
      package:
        exclude:
          - ./**
        include:
          - "handler.js"
          - "logger.js"
      layers:
        - !Ref SqsWorkerLibLambdaLayer
        
        
        How to deploy only trackuserjobs ? sls deploy ???

**Comment by mnapoli** [comment]
You run `sls deploy` to deploy everything.

**Reply by kamalnathlp** [reply]
@mnapoli Found the solution `sls deploy function -f  <functionName>Worker --stage <stage>`
